src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingStonemason.java

```diff
@@ -123,8 +126,12 @@ else if (storage.getPrimaryOutput().getItem() == Items.FLOWER_POT)
                 if (stack.getItem() instanceof BlockItem)
                 {
                     final Block block = ((BlockItem) stack.getItem()).getBlock();
-                    if (block.isIn(Tags.Blocks.STONE) || block.isIn(Tags.Blocks.COBBLESTONE) || block.isIn(BlockTags.STONE_BRICKS))
-                    {
+                    if (
+                            block.isIn(Tags.Blocks.STONE)
+                                    || block.isIn(Tags.Blocks.COBBLESTONE)
+                                    || block.isIn(BlockTags.STONE_BRICKS)
+                                    || block.asItem().getRegistryName().getPath().contains("smooth_stone")
+                    ) {
```

Comment:
We also prefer braces to be on new lines please, and the closing bracket should be on the previous line =D